scripts/scil_filter_tractogram_anatomically.py

```diff
@@ -246,28 +323,42 @@ def main():
         ctx_mask == int(1))] = 1
 
     # Filter tractogram
-    loop_length_CSF_WM_sft , _ = filter_grid_roi(loop_length_CSF_sft,
+    loop_length_csf_wm_sft , _ = filter_grid_roi(loop_length_csf_sft,
         freesurfer_mask, 'both_ends', False)
```

Comment:
PEP 8: continuation line under-indented for visual indent

Reply:
Done in dbd0b2a.